.github/workflows/submitProduction.yml

```diff
@@ -48,9 +48,6 @@ jobs:
       - run: yarn setup && yarn build:freighter-api && yarn build:extension:production --env AMPLITUDE_KEY="${{ secrets.AMPLITUDE_KEY }}" SENTRY_KEY="${{ secrets.SENTRY_KEY }}"
       - name: Install zip
         uses: montudor/action-zip@0852c26906e00f8a315c704958823928d8018b28 #v1.0.0
-      - name: Zip extension build
-        run: zip -qq -r ./build.zip *
-        working-directory: ./extension/build
```

Comment:
this zip file was only used for FF. I'm using the Chrome zip build we generate further down to upload to the GH release page